src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetailsForm.tsx

```diff
@@ -450,7 +511,9 @@ const FulfillmentDetailsFormLayout = (
                   autoCapitalize="characters"
                   autoCorrect="off"
                   value={values.attributes.postalCode}
-                  onChange={trackAutoCompleteEdits("postalCode", handleChange)}
+                  onChange={withBackToFulfillmentDetails(
+                    trackAutoCompleteEdits("postalCode", handleChange)
+                  )}
```

Comment:
`useBackToFulfillmentDetails` hook behavior now wraps specific fields we want to track.